dns/exported/exported.go

```diff
@@ -0,0 +1,22 @@
+package exported
+
+// Entry is the entry used in the DNS.
+type Entry interface {
+	GetOwners() [][]byte
+	GetContent() []byte
+}
+
+// MsgPreCommitEntry defines the msg interface used to send a pre-commit of a
+// registration in the DNS.
+type MsgPreCommitEntry interface {
+	GetKey() []byte
```

Comment:
What is `Key` here? Is it the lookup on the DNS? Shouldn't it be string then

Reply:
For readability I guess it could be a string. The KVStore maps []byte to []byte. 

On separate note, I think it should be possible maybe the IPLD hash should be apart of the mapping. I could see it being useful trying to lookup an owner when given just a hash